Sources/Workspace/Workspace.swift

```diff
@@ -420,6 +441,7 @@ public class Workspace {
         self.repositoryManager = repositoryManager
 
         self.checkoutsPath = self.dataPath.appending(component: "checkouts")
+        self.artifactsPath = self.dataPath.appending(component: "artifacts")
```

Comment:
can you move this to be inside the checkouts directory.

Reply:
That will clash with a SwiftPackage named `artifacts`, won't it?

Reply:
Ah because we dropped the suffix in the names. I guess this path is also fine.